parm/archive/master_gdas.yaml.j2

```diff
@@ -33,6 +33,7 @@ datasets:
     # Wave products
     {% filter indent(width=4) %}
 {% include "gdaswave.yaml.j2" %}
+{% include "gdaswave_restart.yaml.j2" %}
```

Comment:
The `gdaswave_restart.yaml.j2` file should be split into `gdaswave_restarta.yaml.j2` and `gdaswave_restartb.yaml.j2` and each should explicitly grab the file needed.  We can make this a follow-up issue.  For now, how about archiving `gdaswave_restart.yaml.j2` if `save_warm_start_forecast` or `save_warm_start_cycled` is True?  This could be added near line 90 of master_gdas.yaml.j2:

```jinja
    {% if save_warm_start_cycled or save_warm_start_forecast %}
        {% if DO_WAVE %}
            # TODO: The wave restart files should be split into group a and b
            {% filter indent(width=4) %}
{% include "gdaswave_restart.yaml.j2" %}
            {% endfilter %}
        {% endif %}
    {% endif %}
```

Reply:
I see.  Thanks for clarifying the HAFS downstream component.  You can leave this as-is.